ML_20th_Century.ipynb

```diff
@@ -0,0 +1,2104 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Machine Learning for the 20th century: <br> Can a United Kingdom database from 1987 classify a glass cosmetics jar from the 1930s that might have belonged to Amelia Earhart?\n",
+    "Author: Joe Cerniglia <br>\n",
+    "Reviewers: <br>\n",
+    "Date: March 18, 2022 <br><br>\n",
+    "\n",
+    "## Background\n",
```

Comment:
Suggestions for making the copy a bit more concise in the Background section:

In summer 2010, researchers from The International Group for Historic Aircraft Recovery excavated fragments of a small, semi-opaque cosmetic jar from the Pacific island of Nikumaroro. They were searching for evidence to support their hypothesis that Amelia Earhart and Fred Noonan, her navigator, perished there in 1937 whilst on their mission to circumnavigate the globe by air along the equator, its widest point.

My personal interest in the island's glass artefacts led me to become the team's resident glass expert, and in partnership with archaeologists Thomas King and Bill Lockhart, and chemist Greg George, we produced a [paper](https://www.academia.edu/40823470/A_Freckle_In_Time_or_a_Fly_in_the_Ointment) that summarized our findings. By then, our work on the jar had already generated some [press](https://www.nbcnews.com/id/wbna47623025).

As we continued our research in 2013, I worked with [EAG Laboratories](https://www.eag.com/) to determine the chemical composition of the artefact jar, and a 'sibling' glass jar that I had purchased on eBay and had named the clear facsimile. While the two jars were similar in most respects, importantly the clear facsimile jar was transparent. As EAG's [report](https://tighar.org/Projects/Earhart/Archives/Research/ResearchPapers/freckleintime/Document_02_Facsimileclearjarreport.pdf) shows, while both jars were manufactured by the Hazel-Atlas company of Wheeling, West Virginia, the two jars turned out to have very different chemical compositions.